src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss

```diff
@@ -113,13 +113,9 @@ div {
 @media (min-width: 1024px) {
   .eco-news_list-view-wrp {
     height: 204px;
-    -ms-grid-columns: 359px auto;
-    grid-template-columns: 359px auto;
-  }
-
-  .eco-news_list-content-title {
-    font-size: 21px;
-    line-height: 26px;
+ bugfix/#1897-fix-incorrect-size-of-the-news-item's-title
+    -ms-grid-columns: 299px auto;
+    grid-template-columns: 299px auto;
```

Comment:
299px, please double-check new design, so we will be sure about this values

Reply:
![image](https://user-images.githubusercontent.com/24696596/113354756-4f29cd00-9348-11eb-8a10-335842c4529e.png)

link to new mockup
https://www.figma.com/file/092AhCGadc1Hq8nAysxmYB/ITA-Greencity-%D0%A3%D0%91%D0%A1?node-id=12671%3A6400